presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -2,6 +2,12 @@
 Date and Time Functions and Operators
 =====================================
 
+Use the date and time functions and operators, detailed in the following, to
```

Comment:
We don't normally give directives in the documentation. Instead, phrase this as

```
These functions and operators operate on :ref:`date and time data types <date-time-data-types>`.

Reply:
I would be more approachable to address the user .. do you still want me to rejig?

Reply:
Yes. We want to keep the style consistent.